### ISSUE TITLE
What's the difference between "Qwen2.5-MATH-7B-PRM800K" and "Our PRM-7B" ？

### ISSUE THREAD

**Issue by HCHCXY** [CJReinforce/PURE#1]

**Comment by CJReinforce** [comment]
In the README, both `Qwen2.5-MATH-7B-PRM800K` and `Our PRM-7B` refer to the same PRM [here](https://huggingface.co/jinachris/Qwen2.5-Math-7B-PRM800K). The former name shows the base model and training data set of the PRM. 

We haven't come up with a good name for our PRM yet, because the most obvious name, `PURE-PRM`, is referred to the LLM that is fine-tuned using only PRM in the README. This led us to use two other names (`Qwen2.5-MATH-7B-PRM800K` and `Our PRM-7B`) to refer to our PRM.

**Comment by HCHCXY** [comment]
<img width="793" alt="Image" src="https://github.com/user-attachments/assets/716594be-a6ac-4cee-91b0-9dcf35e1c0d7" />
I'm a little confused about why the statistics of last 2 columns show a difference.

**Comment by CJReinforce** [comment]
ah, I see. The model in the second-to-last row refers to the [PRM from Qwen team](https://huggingface.co/Qwen/Qwen2.5-Math-7B-PRM800K), which is trained based on Qwen2.5-math-7b-Instruct model. The model in the last row refers to [our PRM](https://huggingface.co/jinachris/Qwen2.5-Math-7B-PRM800K) that is trained based on Qwen2.5-math-7b, because all the baselines and our LLM are based on that model.

It seems that I need to give our PRM a suitable name to distinguish them as soon as possible.

**Comment by HCHCXY** [comment]
Thanks for your time and explanation!

**Comment by HCHCXY** [comment]
I check "The Lessons of Developing Process Reward Models in Mathematical Reasoning". The
